### PR TITLE
Add linguist-tools to dependencies

### PR DESCRIPTION
**Pull request by pavel-the-best**
Closes: https://bugs.gentoo.org/730428
Signed-off-by: Pavel Kalugin <paul.kalug@gmail.com>

### CODE REVIEW

File: media-gfx/flameshot/flameshot-0.6.0.ebuild [hunk 1]
```diff
@@ -22,6 +22,7 @@ DEPEND="
 	dev-qt/qtsvg:5
 	dev-qt/qtnetwork:5
 	dev-qt/qtdbus:5
+	dev-qt/linguist-tools:5
```
Comment (juippis) [inline media-gfx/flameshot/flameshot-0.6.0.ebuild:25]: With EAPI-7 this is a `BDEPEND`. Can you update your commit summary to match our requirements?
https://www.gentoo.org/glep/glep-0066.html#commit-messages

Comment (pavel-the-best) [reply media-gfx/flameshot/flameshot-0.6.0.ebuild:25]: It seems, that commit summary already matches the requirements

Comment (a17r) [reply media-gfx/flameshot/flameshot-0.6.0.ebuild:25]: > It seems, that commit summary already matches the requirements

not at all. just read the link provided to you.

Comment (pavel-the-best) [reply media-gfx/flameshot/flameshot-0.6.0.ebuild]: Oh, I see. Thanks, I didn't know about `repoman commit`.

File: media-gfx/flameshot/flameshot-0.6.0.ebuild [hunk 2]
```diff
@@ -24,6 +24,10 @@ DEPEND="
 	dev-qt/qtdbus:5
 	sys-apps/dbus
 "
+BDEPEND="
+	${DEPEND}
+	dev-qt/linguist-tools:5
+"
```
Comment (juippis) [inline media-gfx/flameshot/flameshot-0.6.0.ebuild:30]: `BDEPEND` is a split from `DEPEND`, so `BDEPEND="${DEPEND}"` is wrong by design :)

Simple `BDEPEND="dev-qt/linguist-tools:5"` works here. Rest looks good.

Comment (pavel-the-best) [reply media-gfx/flameshot/flameshot-0.6.0.ebuild]: Good now?

**Pull request closed without merging**